package.js

```diff
@@ -1,30 +1,27 @@
 Package.describe({
   summary: 'Style with attitude. Sass and SCSS support for Meteor.js.',
-  version: '4.17.0-rc.0',
+  version: '4.17.0',
   git: 'https://github.com/Meteor-Community-Packages/meteor-scss.git',
   name: 'fourseven:scss',
 });
 
 Package.registerBuildPlugin({
   name: 'compileScssBatch',
-  use: [
-    'caching-compiler@2.0.0',
-    'ecmascript@0.16.9'
-  ],
+  use: ['caching-compiler@2.0.1', 'ecmascript@0.16.10'],
   sources: ['plugin/compile-scss.js'],
   npmDependencies: {
-    "node-sass": '9.0.0',
-    "@babel/runtime": "7.24.5"
+    'node-sass': '9.0.0',
+    '@babel/runtime': '7.26.0',
   }
-})
+});
 
 Package.onUse(function(api) {
-  api.versionsFrom(['2.8.0', '3.0.1']);
+  api.versionsFrom(['3.0.1']);
   api.use('isobuild:compiler-plugin@1.0.0');
 });
 
 Package.onTest(function(api) {
-  api.versionsFrom(['2.8.0', '3.0.1']);
```

Comment:
I'm a bit uncomfortable to remove support for Meteor 2 in a feature patch.
#323 is doing a major version bump, so that one is going to take care of things there.

Any way we can keep 2.8 as a target?

Reply:
Agree. Pushed `2.8.0` support back to this PR